src/main/java/com/worldheatmap/WorldHeatmapPlugin.java

```diff
@@ -411,10 +410,6 @@ public void onNpcLootReceived(final NpcLootReceived npcLootReceived) {
      * Autosave the heatmap file/write the 'TYPE_A' and 'TYPE_B' heatmap images if it is the correct time to do each respective thing according to their frequencies
      */
     private void autosaveRoutine() {
-        File heatmapsFile = new File(HEATMAP_FILES_DIR, mostRecentLocalUserID + ".heatmaps");
-        File typeAImageFile = new File(HEATMAP_IMAGE_DIR, mostRecentLocalUserID + "_" + HeatmapNew.HeatmapType.TYPE_A + ".tif");
-        File typeBImageFile = new File(HEATMAP_IMAGE_DIR, mostRecentLocalUserID + "_" + HeatmapNew.HeatmapType.TYPE_B + ".tif");
-
```

Comment:
Not necessary to create the files when it's not autosave anyway. I moved it to inside the if / else